### PR TITLE
Created alerts and badges

### PR DESCRIPTION
**Pull request by aryanandarnav13**

### CODE REVIEW

**Review by sks-21 — APPROVED** [review]
The code looks nice. Good work !

**Comment by aryanandarnav13** [comment]
> The code looks nice. Good work !

Thanks mahn

File: css/alert.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
+.alertd {
+    padding: 1rem;
+    background-color: rgb(248, 66, 66);
+    color: white;
+    margin:1rem;
+  }
+  .alerts {
+    padding: 1rem;
+    background-color: rgb(116, 243, 116);
```
Comment (animeshgarai09) [inline css/alert.css:9]: Put all the colors in CSS variables which is used repeatedly.

File: index.html [hunk 2]
```diff
@@ -32,9 +35,40 @@ <h2>Avatar</h2>
...
+      <button class="badge-btn "> <i class="fas fa-home"></i> <span class="badge-text">2</span> </button>
+      <button class="badge-btn "> <i class="fas fa-pen"></i> <span class="badge-text">3</span> </button>
+      <button class="badge-btn "> <i class="fas fa-car"></i> <span class="badge-text">4</span> </button>
+      <h4>Badge on Avatars</h4>
+      <div class="badge">
+        <img class="avatar" src="assets/img_avatar.png" alt=""/>
+        <span class="badge-status1 badge-status"></span>
+      </div>
+      <div class="badge">
+        <img class="avatar" src="assets/img_avatar.png" alt=""/>
```
Comment (animeshgarai09) [inline index.html:65]: Fill alt value

**Review by animeshgarai09 — CHANGES_REQUESTED** [review]
Rest looks good, a few changes needed

**Comment by yash9263** [comment]
Everything looks good to me

**Review by yash9263 — APPROVED** [review]

File: css/alert.css (hunk 1 above)
Comment (aryanandarnav13) [reply css/alert.css:9]: Okay, Will work on it.

File: index.html (hunk 2 above)
Comment (aryanandarnav13) [reply index.html:65]: Yes , thanks

**Pull request merged**